test/MixedLanguage/tensor.cu

```diff
@@ -0,0 +1,121 @@
+/*******************************************************************************
+ * Copyright (c) 2022 - 2024 NVIDIA Corporation & Affiliates.                  *
+ * All rights reserved.                                                        *
+ *                                                                             *
+ * This source code and the accompanying materials are made available under    *
+ * the terms of the Apache License 2.0 which accompanies this distribution.    *
+ ******************************************************************************/
+
+// REQUIRES: nvcc
+
+// RUN: (nvcc -I runtime -c -Xcompiler -fPIC %cpp_std %p/tensor.cu -o %t.o && \
+// RUN: nvq++ %cpp_std --enable-mlir %s %t.o -L `dirname $(which nvcc)`/../lib64 -lcudart -o %t && echo "Success") | \
```

Comment:
nvq++ shouldn't be compiling .cu files, right?

Reply:
right, copy-and-paste gone wrong:)

Reply:
done, thanks!